src/web/app/pages-student/student-home-page/student-home-page.component.ts

```diff
@@ -1,44 +1,36 @@
 import { Component, OnInit } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
+import moment from 'moment-timezone';
+import { FeedbackSessionsService } from '../../../services/feedback-sessions.service';
 
 import { HttpRequestService } from '../../../services/http-request.service';
 import { StatusMessageService } from '../../../services/status-message.service';
+import { TimezoneService } from '../../../services/timezone.service';
+import {
+  Course,
+  Courses,
+  FeedbackSession,
+  FeedbackSessionPublishStatus,
+  FeedbackSessions,
+  FeedbackSessionSubmissionStatus,
+  HasResponses,
+} from '../../../types/api-output';
 import { ErrorMessageOutput } from '../../error-message-output';
 
-interface SessionInfoMap {
+interface StudentCourse {
+  course: Course;
+  feedbackSessions: StudentSession[];
+}
+
+interface StudentSession {
+  session: FeedbackSession;
   endTime: string;
   isOpened: boolean;
   isWaitingToOpen: boolean;
   isPublished: boolean;
   isSubmitted: boolean;
```

Comment:
export `FeedbackSessionPublishStatus` to the template so you don't need `isPublished`, `isWaitingToOpen` and `isOpened `?